arangod/VocBase/vocbase.cpp

```diff
@@ -1439,24 +1438,6 @@ arangodb::Result TRI_vocbase_t::renameCollection(
     return TRI_ERROR_NO_ERROR;
   }
 
-  if (!doOverride) {
```

Comment:
is doOverride never used ?

Reply:
it is, but only from V8 handler, so I moved it to Methods::Collections